src/SFA.DAS.EmployerIncentives.Data.UnitTests/ApprenticeApplicationDataRepository/WhenGetListCalled.cs

```diff
@@ -2222,5 +2223,311 @@ public async Task Then_the_employment_check_is_not_set_if_the_validation_result_
             application.FirstPaymentStatus.EmploymentCheckPassed.Should().BeNull();
             application.SecondPaymentStatus.EmploymentCheckPassed.Should().BeNull();
         }
+
+        [TestCase(false, false, true)]
+        [TestCase(true, false, true)]
+        [TestCase(false, true, false)]
+        [TestCase(true, true, true)]
+        public async Task When_IsInLearning_called_should_return_true_if_IsInLearning_is_true_or_false_and_there_is_an_override(bool inLearning, bool hasExpired, bool expectation)
```

Comment:
The rest of the test names in this class are prefixed with Then_, can these new ones be renamed for consistency?